cmd/ping/ping.go

```diff
@@ -31,12 +31,16 @@ Examples:
 
     # Send a privileged raw ICMP ping
     sudo ping --privileged www.google.com
+
+    # Send ICMP messages with a 100-byte payload
+    ping -s 100 1.1.1.1
 `
 
 func main() {
 	timeout := flag.Duration("t", time.Second*100000, "")
 	interval := flag.Duration("i", time.Second, "")
 	count := flag.Int("c", -1, "")
+	size := flag.Int("s", 16, "")
```

Comment:
Default to traditional ping 56?

Reply:
I was thinking of keeping it at the current default for now and then opening a separate PR to propose we change it to the traditional default both here and in the following line:

https://github.com/go-ping/ping/blob/2c1f7d31c4d0a9d40be1106d8e0274ee718738c9/ping.go#L93